L1Trigger/L1TTrackMatch/python/L1TkElectronTrackProducer_cfi.py

```diff
@@ -0,0 +1,89 @@
+import FWCore.ParameterSet.Config as cms
+
+L1TkElectrons = cms.EDProducer("L1TkElectronTrackProducer",
+	label = cms.string("EG"),	# labels the collection of L1TkEmParticleProducer that is produced.
```

Comment:
please use some kind of consistent indentation, this is a mess

Reply:
done